ui/packages/shared/profile/src/Callgraph/utils.ts

```diff
@@ -94,28 +93,27 @@ export const jsonToDot = ({
   colorRange: [string, string];
 }): string => {
   const {nodes, edges} = graph;
-  const cumulatives = nodes.map((node: CallgraphNode) => node.cumulative);
-  const cumulativesRange = d3.extent(cumulatives).map(value => Number(value));
-
+  const cumulatives = edges.map((edge: CallgraphEdge) => Number(edge.cumulative));
+  const cumulativesRange = d3.extent(cumulatives) as [number, number];
   const colorScale = d3
     .scaleSequentialLog(d3.interpolateBlues)
     .domain(cumulativesRange)
     .range(colorRange);
-  const colorOpacityScale = d3.scaleSequentialLog().domain(cumulativesRange).range([0.2, 1]);
-  const boxWidthScale = d3
-    .scaleLog()
-    .domain(cumulativesRange)
-    .range([DEFAULT_NODE_HEIGHT, DEFAULT_NODE_HEIGHT + 40]);
+  const colorOpacityScale = d3.scaleLinear().domain(cumulativesRange).range([0.5, 1]);
+  // const boxWidthScale = d3
+  //   .scaleLog()
+  //   .domain(cumulativesRange)
+  //   .range([DEFAULT_NODE_HEIGHT, DEFAULT_NODE_HEIGHT + 40]);
```

Comment:
Same with the comments here

Reply:
yeah, kept this here in case we wanted to differentially size the callgraph nodes. That being said, I think we can delete these lines for now and add back in if/when we decide to change the node sizing strategy.